### PR TITLE
feat: add nestjs-file-storage and angular-file-storage libs

### PR DESCRIPTION
**Pull request by maxmousse**

### CODE REVIEW

**Comment by maxmousse** [comment]
nestjs-file-storage and angular-file-storage libraries work

Some features are still missing but can be added later:
-> CRON to prune out of date temporary files on file storage
-> File storage updload component for front-end
-> Unit tests 
-> Documentation in readme
-> In nestjs, separate cli in its own module to cherry pick imports

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#---------------------------------------
+# File storage keys
+#---------------------------------------
+TRACTR_FILE_STORAGE_ACCESS_KEY=minio
+TRACTR_FILE_STORAGE_SECRET_KEY=password
```
Comment (floross) [inline .env.example]: Add a newline

File: libs/angular/file-storage/src/lib/file-storage.module.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    );
+    const errors = validateSync(configurationDto, {
+      skipMissingProperties: false,
+    });
+
+    if (errors.length > 0) {
+      throw new Error(
+        `Invalid configuration for FileStorageModule: \n\n${errors.join('\n')}`,
+      );
+    }
```
Comment (floross) [inline libs/angular/file-storage/src/lib/file-storage.module.ts]: You can use `transformAndValidate` from `@tractr/common`

File: libs/hapify/templates/prisma/package.json [hunk 3]
```diff
@@ -7,6 +7,10 @@
     "url": "https://github.com/tractr/stack"
   },
   "license": "UNLICENSED",
+  "peerDependencies": {
+    "@prisma/client": "^2.24.1",
+    "prisma": "^2.24.1"
```
Comment (floross) [inline libs/hapify/templates/prisma/package.json:12]: Are you sure about the version of prisma ?

**Review by floross — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (maxmousse) [reply .env.example]: Done

File: libs/hapify/templates/prisma/package.json (hunk 3 above)
Comment (maxmousse) [reply libs/hapify/templates/prisma/package.json:12]: Done

**Pull request merged**